packages/remark-language-server/readme.md

```diff
@@ -0,0 +1,207 @@
+# remark-language-server
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+A language server to lint and format markdown files with **[remark][]**.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [CLI](#cli)
+*   [Compatibility](#compatibility)
+*   [Contribute](#contribute)
+*   [Sponsor](#sponsor)
+*   [License](#license)
+
+## What is this?
+
+This package is a [language server][] which can lint and format markdown files
+using remark.
+
+You can choose from the 150+ plugins that already exist or make your own.
```

Comment:
Perhaps a small clarification that it's just another layer but the config can be shared between the remark CLI and editor integration.
 
```suggestion
The language server can take in your [remark][] configuration file so you can use the settings and all the plugins as you're used to. 
```

Reply:
I actually copied the readme from `remark-cli` and modified it. The original line comes from there, so I’m not sure if it should be removed here.

Reply:
Maybe remove this? The previous and next sentence seem to explain “What is this”. This line seems to be about plugins, which aren’t introduced or explained in surrounding prose.